app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,7 @@
+module StaticPagesHelper
+  
+  def user_can_edit?
+    current_user && current_user.is_privileged? || current_user.membership_type != 'Basic' 
```

Comment:
do we need the user_signed_in? ? I guess not - current_user nil check will handle that ...